src/app/components/date-range-picker/index.html

```diff
@@ -0,0 +1,262 @@
+<sky-demo-page
+  pageTitle="Date range picker"
+  skyuxModule="SkyDateRangePickerModule"
+  packageName="@skyux/datetime"
+>
+  <sky-demo-page-summary>
+    The date range picker component creates a text input with a dropdown for users to select a date range from a set of well-known options. This includes options that enable users to specify dates to create their own date range. The text input is a composite component of up to two <stache-code>sky-datepicker</stache-code> components.
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties
+    sectionHeading="Date range picker component properties"
+  >
+    <sky-demo-page-property
+      propertyName="label"
+    >
+      Specifies a label for the date range picker. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="calculatorIds"
```

Comment:
Would it be helpful to explain what a "calculator" is before we get to this property? They're the fundamental elements of a date-range picker, so maybe they deserve a quick explanation? At first glance, I didn't inertially understand what they were until I read saw it was used.

Reply:
More info around "calculators": https://github.com/blackbaud/skyux2-docs/issues/261#issuecomment-488773180

Reply:
Thanks for the updates! That reads much better :)